packages/field-plugin/helpers/vite/src/sandbox.ts

```diff
@@ -0,0 +1,64 @@
+import * as querystring from 'querystring'
+import { bold } from './utils/text'
+import { arrows } from './utils/arrows'
+import { load, fileExists, MANIFEST_FILE_NAME } from './manifest'
+import type { Manifest } from './manifest'
+
+export const SANDBOX_BASE_URL = `https://plugin-sandbox.storyblok.com/field-plugin`
+
+export type SandboxQueryParams = {
+  url: string
+  manifest: Manifest | null
+}
+
+export const buildQueryString = (params: SandboxQueryParams) => {
+  const queryParams: any = {
```

Comment:
I think I was receiving an error before if I didn't specify it @eunjae-lee. However, I tried it here again and couldn't reproduce it. So, I removed it here. 

Thanks 🎯

Reply:
Thanks @demetriusfeijoo. Next time I'd appreciate if you don't resolve these conversations! I suggest something to your PR, you reply, and then I confirm it and I can resolve them. If you resolve them, it's harder for me to find all the latest comments of yours :)

Reply:
Surely, @eunjae-lee. 🎯
Sorry for that. 

I felt it in my skin right now when trying to reply to you here. I knew you had commented because I saw it in my email inbox, but it wasn't easy to find here. 😅

Reply:
Haha no worries ! 😂